pynest/examples/eprop_plasticity/online_eprop_supervised_classification_evidence-accumulation.py

```diff
@@ -0,0 +1,815 @@
+        "delay_in_rec": 1,  # connection delay between input and recurrent neurons
+        "delay_rec_out": 0,  # connection delay between recurrent and output neurons
+        "delay_out_norm": 0,  # connection delay between output neurons for normalization
+        "extension_sim": 3,  # extra time step to close right-open simulation time interval in Simulate()
+    }
+)
+
+steps["delays"] = steps["delay_in_rec"] + steps["delay_rec_out"] + steps["delay_out_norm"]  # time steps of delays
+
+steps["total_offset"] = steps["offset_gen"] + steps["delays"]  # time steps of total offset
+
+steps["sim"] = steps["task"] + steps["total_offset"] + steps["extension_sim"]  # time steps of simulation
+
+duration = {"step": 1.0}  # ms, temporal resolution of the simulation
+
+duration.update({key: value * duration["step"] for key, value in steps.items()})  # ms, durations
+
+# %% ###########################################################################################################
+# Set up simulation
+# .................
+# As last step of the setup, we reset the NEST kernel to remove all existing NEST simulation settings and
+# objects and set some NEST kernel parameters, some of which are e-prop-related.
+
+params_setup = {
+    "eprop_reset_neurons_on_update": True,  # if True, reset dynamic variables at start of each update interval
+    "eprop_update_interval": duration["sequence"],  # ms, time interval for updating the synaptic weights
+    "print_time": False,  # if True, print time progress bar during simulation, set False if run as code cell
+    "resolution": duration["step"],
+    "total_num_virtual_procs": 1,  # number of virtual processes, set in case of distributed computing
+}
+
+####################
+
+nest.ResetKernel()
+nest.set(**params_setup)
+
+# %% ###########################################################################################################
+# Create neurons
+# ~~~~~~~~~~~~~~
+# We proceed by creating a certain number of input, recurrent, and readout neurons and setting their parameters.
+# Additionally, we already create an input spike generator and an output target rate generator, which we will
+# configure later. Within the recurrent network, alongside a population of regular neurons, we introduce a
+# population of adaptive neurons, to enhance the network's memory retention.
+
+n_in = 40  # number of input neurons
+n_ad = 50  # number of adaptive neurons
+n_reg = 50  # number of regular neurons
+n_rec = n_ad + n_reg  # number of recurrent neurons
+n_out = 2  # number of readout neurons
+
+model_nrn_reg = "eprop_iaf"
+
+params_nrn_out = {
+    "C_m": 1.0,  # pF, membrane capacitance - takes effect only if neurons get current input (here not the case)
+    "E_L": 0.0,  # mV, leak / resting membrane potential
+    "eprop_isi_trace_cutoff": 10,  # cutoff of integration of eprop trace between spikes
+    "I_e": 0.0,  # pA, external current input
+    "loss": "mean_squared_error",  # loss function
```

Comment:
```suggestion
```

Reply:
Could you please elaborate on this change? The script would work with your change as `"means_squared_error"` is the default, but wouldn't it be better to make it explicit?

Reply:
Hold on, doesn't complain if you include that line?

Please check:
9891267b6cb0258417b830e5f1dadd7ea66c03c8

Reply:
I suggest my suggested change is not added to this PR but the change is added or discussed separately.